### PR TITLE
fix(deps): adds core dep in client to right place

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "@textile/context": "^0.1.0-rc.0",
     "@textile/multiaddr": "^0.0.4-rc.0",
     "@textile/threads-client-grpc": "^0.1.16",
+    "@textile/threads-core": "^0.1.6-rc.0",
```
Comment (carsonfarmer) [inline packages/client/package.json:43]: Is this actually used at run time? Must have missed something...

Comment (andrewxhill) [reply packages/client/package.json:43]: https://github.com/textileio/js-threads/blob/master/packages/client/src/index.ts#L10

Comment (carsonfarmer) [reply packages/client/package.json:43]: Yeh, its actually this line (which I did miss): https://github.com/textileio/js-threads/blob/master/packages/client/src/index.ts#L66

Comment (carsonfarmer) [reply packages/client/package.json:43]: Good catch!

**Pull request merged**